build.gradle

```diff
@@ -71,6 +72,51 @@ subprojects {
     apply plugin: 'idea'
     apply plugin: 'eclipse'
     apply plugin: 'java'
+    apply plugin: 'com.chrisgahlert.gradle-dcompose-plugin'
+
+    // Using gradle-dcompose-plugin to build/push docker images
+    dcompose {
+        if (rootProject.hasProperty('dockerRegistry')) {
```

Comment:
Here, if I'm reading right, we do nothing if the `dockerRegistry` isn't defined.  Don't we want to just not publish here?  That is, we'd have everything build except the client closures?